R/themes.R

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8-unix -*-
+library(tidyverse)
+
+theme_maps <- theme(
+  axis.ticks = element_blank(),
+  axis.text = element_blank(),
+  axis.line = element_blank(),
+  panel.background = element_blank(),
+  panel.border = element_blank(),
+  plot.title = element_text(size = 10, face = "bold"),
+  plot.title.position = "panel",
+  plot.subtitle = element_blank(),
+  legend.background = element_rect(fill = "transparent"),
+  legend.title = element_text(size = 10),
+  legend.text = element_text(size = 9),
+  legend.justification = c(0, 1),
+  legend.position = c(0, 1),
+  plot.margin = unit(c(0.25, 0.25, 0.25, 0.25), "cm")
+)
```

Comment:
This `theme_maps` will most likely be replaced later with [theme_mal_map](https://github.com/HSLdevcom/mal-effect-calculations/blob/da275efe220c355ddc0441cc47e1499553732f4b/Basemaps/functions_map.R#L75) function. I can do it later or I can share you the input files so you can test it too?

Reply:
@atte-wsp Yes, they are in the same file. [geom_basemap](https://github.com/HSLdevcom/mal-effect-calculations/blob/d6a7c302ba89fff0a4a3a883ee6921c2c7fa59cc/scripts/basemap/functions_map.R#L16) adds waters, rails etc. and [annotate_map](https://github.com/HSLdevcom/mal-effect-calculations/blob/d6a7c302ba89fff0a4a3a883ee6921c2c7fa59cc/scripts/basemap/functions_map.R#L36) adds titles. You can see them in use [in this basemap](https://github.com/HSLdevcom/mal-effect-calculations/blob/main/scripts/basemap/map.R). You can view the final map in Sharepoint `Vaikutusten arviointi/MAL 2019 mittareiden työstö/map_luonnos_2021-03-17.png`.